src/mainboard/protectli/vault_ehl/gpio.h

```diff
@@ -0,0 +1,1664 @@
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U1, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U2 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U2, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U3 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U3, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U4 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U4, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U5 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U5, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U6 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U6, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U7 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U7, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U8 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U8, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U9 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U9, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U10 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U10, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U11 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U11, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U12 - ISI_CHX_OKNOK_0 */
+	/* DW0: 0x44000700, DW1: 0x00001000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	PAD_CFG_NF(GPP_U12, DN_20K, DEEP, NF1),
+
+	/* GPP_U13 - ISI_CHX_OKNOK_1 */
+	/* DW0: 0x44000700, DW1: 0x00001000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	PAD_CFG_NF(GPP_U13, DN_20K, DEEP, NF1),
+
+	/* GPP_U14 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U14, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U15 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_U15, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_U16 - ISI_OKNOK_0 */
+	/* DW0: 0x44600702, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) | (1 << 1) - IGNORED */
+	PAD_CFG_NF(GPP_U16, NONE, DEEP, NF1),
+
+	/* GPP_U17 - ISI_OKNOK_1 */
+	/* DW0: 0x44600702, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) | (1 << 1) - IGNORED */
+	PAD_CFG_NF(GPP_U17, NONE, DEEP, NF1),
+
+	/* GPP_U18 - ISI_ALERT_N */
+	/* DW0: 0x44600702, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) | (1 << 1) - IGNORED */
+	PAD_CFG_NF(GPP_U18, NONE, DEEP, NF1),
```

Comment:
Pads are NC, BTW GPP_U19 is missing.

Reply:
Good you noticed that U19 is missing. In inteltool it had `GP_U19` set as a name, then intelp2m didn't match that name. Fixed that in inteltool upstream.

Reply:
Done, also added that U19 with PAD_NC.